src/components/Pokedex.tsx

```diff
@@ -14,33 +22,64 @@ const Item = ({ title }: ItemProps) => (
 );
 
 export default function Pokedex({ navigation }: { navigation: any }) {
-    return (
-        <View style={styles.container}>
-            <Text>Open up App.tsx to start working on your app!</Text>
-
-            <FlatList
-                data={pokemonList}
-                renderItem={({ item }) =>
-                    <TouchableHighlight
-                        style={styles.touchable}
-                        activeOpacity={0.6}
-                        underlayColor="#DDDDDD"
-                        onPress={() => navigation.navigate('PokemonDetails',
-                            {
-                                name: item.name,
-                                uri: item.url,
-                            }
-                        )}>
-                        <Item title={item.name} />
-                    </TouchableHighlight>
-
-                }
-                keyExtractor={item => item.id}
-            />
-
-            <StatusBar style="auto" />
-        </View>
-    );
+    const pokemonList = require("../../assets/kanto.json");
+
+    const fetchProjects = async ({ pageParam = 0 }) => {
+        const res = await fetch('https://pokeapi.co/api/v2/pokemon/?offset=' + pageParam + '&limit=20')
```

Comment:
Generally should prefer template literals over concatenation for readability purposes
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals